kafka-ui-react-app/src/components/Connect/Details/Overview/Overview.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { Connector } from 'generated-sources';
-import TagStyled from 'components/common/Tag/Tag.styled';
+import * as C from 'components/common/Tag/Tag.styled';
```

Comment:
Why C ?)
We usually use like this
import * as S from '.styled';

Reply:
I think that we use S when we import styles for tis particular component, it's own styles. Here I'm importing common component, so that's 'C' alias